aoe4/formatters.js

```diff
@@ -124,13 +129,21 @@ class NightbotDefaultFormatter {
 
     const rank = mode.rank ? `rank #${mode.rank}` : 'unranked';
     const rank_level = mode.rank_level ? formatRankLevel(mode.rank_level) + ', ' : '';
-    var msg = `${player.name} is ${rank} (${rank_level}${mode.rating} Elo), with ${mode.games_count} game${mode.games_count == 1?'':'s'} (${mode.wins_count}-${mode.losses_count} | ${mode.win_rate}%)`;
+    const rating = mode.rating ? ` (${rank_level}${mode.rating} Elo)` : '';
+    var msg = `${player.name} is ${rank}${rating}, with ${mode.games_count} game${mode.games_count == 1?'':'s'}`;
+
+    if (mode.games_count) {
+      msg += ` (${mode.wins_count}-${mode.losses_count} | ${mode.win_rate}%)`;
+    }
 
     if (Number.isInteger(mode.streak)) {
         const streak = parseInt(mode.streak) < 0 ? "losing" : "win";
         msg += `, on a ${Math.abs(mode.streak)}-game ${streak} streak`;
     }
-    msg += ` [last played ${formatAge(mode.last_game_at)} ago]`;
+
+    if (mode.last_game) {
```

Comment:
I think you need to check here for `last_game_at` . coz I get nothing for Beasty: `Beastyqt is rank #1 (C3, 1676 Elo), with 88 games (80-8 | 90.9%), on a 12-game win streak`